modules/react/form-field/lib/Label.tsx

```diff
@@ -75,9 +75,18 @@ const RequiredAstrisk = styled('abbr')({
 // Used inside the fieldset component instead of a label for accessible radio groups.
 // Extra `width: 100%` style is to help IE11 wrap text properly in a top-positioned
 // FormField's legend element.
-const LegendComponent = styled('legend')<LabelProps>(...labelStyles, ({labelPosition}) => ({
-  width: labelPosition === Label.Position.Top ? '100%' : undefined,
-}));
+const LegendComponent = styled('legend')<LabelProps>(...labelStyles, ({labelPosition}) => {
+  if (labelPosition === Label.Position.Top) {
+    return {
+      width: '100%',
+    };
+  }
+
+  return {
+    float: 'left',
```

Comment:
My only concern with `float` is use of a float will decrease rendering performance: https://developers.google.com/web/fundamentals/performance/rendering/avoid-large-complex-layouts-and-layout-thrashing

I don't think this `float` will realistically tank rendering performance of a page and I'm sure many techniques were evaluated to get the correct result in all our supported browsers, but I wanted to make sure `flex` didn't work here first.

Reply:
+1 I prefer flex over float if possible.

Reply:
Okay, after diving into this, I realize that a `<legend>` element inside a `<fieldset>` doesn't support `display: flex`. Fun. I'd like to explore some other solutions, but not in this PR.

Reply:
I too prefer flex, but was unable to achieve the `Label.Position.Left` layout without it. @alanbsmith thankfully this is now  "supported" in modern browsers: https://bugzilla.mozilla.org/show_bug.cgi?id=1230207, https://bugs.chromium.org/p/chromium/issues/detail?id=375693 .... but still some weird edge cases that required the float – "fun" accurately describes this 😂